duckduckgo/iOS
# Disable the packet tunnel provider

Task/Issue URL: https://app.asana.com/0/414235014887631/1204593255118994/f
Tech Design URL:
CC:

**Description**:

This PR removes the packet tunnel provider and disables related code.

The feature is controlled behind a conditional compilation flag, so to turn everything back on you just need to set that value in the release build and add the packet tunnel provider back to the main app target.



**Steps to test this PR**:
1. Check that the app builds and runs locally - it's okay if AppTP shows up in settings, it's still turned on for debug builds so that we can guarantee that it doesn't break without us knowing about it
2. Check that you can archive a build, to validate that the release process isn't broken
3. Do a search for `import NetworkExtension` and make sure there's no code in the main app that is doing anything outside of those imports. Note that not *everything* is disabled behind this flag, just the logic related to networking.
4. Remove the `APP_TRACKING_PROTECTION` value from the main project file settings, and then make sure AppTP doesn't show up in the debug build in settings



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
